src/macro_util.rs

```diff
@@ -61,6 +65,134 @@ impl<T, U> MaxAlignsOf<T, U> {
     }
 }
 
+const _64K: usize = 1 << 16;
+
+// remove this `cfg` when `size_of_val_raw` is stabilized
+#[cfg(__INTERNAL_USE_ONLY_NIGHLTY_FEATURES_IN_TESTS)]
+#[repr(C, align(65536))]
+struct Aligned64kAllocation([u8; _64K]);
```

Comment:
Can we add a test that `assert_eq!(align_of::< Aligned64kAllocation>(), _64K);`?